embedded/sql/row_reader.go

```diff
@@ -297,24 +298,24 @@ func (r *rawRowReader) ScanSpecs() *ScanSpecs {
 	return r.scanSpecs
 }
 
-func (r *rawRowReader) Columns() ([]ColDescriptor, error) {
+func (r *rawRowReader) Columns(ctx context.Context) ([]ColDescriptor, error) {
 	ret := make([]ColDescriptor, len(r.colsByPos))
 	for i := range r.colsByPos {
 		ret[i] = r.colsByPos[i]
 	}
 	return ret, nil
 }
 
-func (r *rawRowReader) colsBySelector() (map[string]ColDescriptor, error) {
+func (r *rawRowReader) colsBySelector(ctx context.Context) (map[string]ColDescriptor, error) {
```

Comment:
same question as above

Reply:
discussed with you, closing this as the context is part of the interface hence included

Reply:
replied above